demos/main/src/main/assets/media.exolist.json

```diff
@@ -506,6 +506,13 @@
         "subtitle_mime_type": "text/x-ssa",
         "subtitle_language": "en"
       },
+      {
+        "name": "SubStation Alpha Style",
+        "uri": "https://storage.googleapis.com/exoplayer-test-media-1/gen-3/screens/dash-vod-single-segment/video-avc-baseline-480.mp4",
+        "subtitle_uri": "https://drive.google.com/uc?export=download&id=16IrvtynQ6-ANRpRX7hU6xEQeFU91LmXl",
+        "subtitle_mime_type": "text/x-ssa",
+        "subtitle_language": "en"
+      },
```

Comment:
Thanks for including this to make testing easier, it's really helpful - FYI I'll drop it before actually merging the PR though.

Reply:
Ah I didn't realise this exercises more styles than just italic/bold - this does actually look like a useful file to add to the demo app (but probably not as part of this PR).

If you're happy to contribute this file, can you open a separate PR containing just the SSA file and I'll take it from there an upload it to our cloud bucket and add an entry to the demo app. Thanks!